cohort_4/playbook/SCHEDULE.md

```diff
@@ -67,26 +67,36 @@ partners.
-| 10:30 AM | [Introduction talk](ACTIVITIES.md#introduction-talk)         |
-| 10:40 AM | [Portfolio workshop](ACTIVITIES.md#portfolio)                |
-| 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                 |
-| 9:50 AM  | [Energizer 3](ACTIVITIES.md#evolution-rock-paper-scissors)   |
-| 2:00 PM  | [Brainstorming](ACTIVITIES.md#brainstorming)                 |
-| 2:20 PM  | [Project time](ACTIVITIES.md#freeform-projects)              |
-| 3:30 PM  | [Links collected for demos](ACTIVITIES.md#links)             |
-| 3:40 PM  | [I have failed circle](ACTIVITIES.md#i-have-failed-activity) |
-| 3:50 PM  | [Demos](ACTIVITIES.md#demos)                                 |
-| 4:25 PM  | [Feedback form](ACTIVITIES.md#feedback-forms)                |
-| 4:30 PM  | [Hack cheer](ACTIVITIES.md#finishing-off-the-day)            |
-| 4:45 PM  | [Post-mortem (ending at 5:30 PM)](ACTIVITIES.md#post-mortem) |
+| Time     | Activity                                                                   |
+| -------- | -------------------------------------------------------------------------- |
+| 8:30 AM  | Standups                                                                   |
+| 9:20 AM  | [Hacker arrival](ACTIVITIES.md#on-the-first-day)                           |
+| 9:30 AM  | Galvanize tour                                                             |
+| 9:40 AM  | [Energizer 1](ACTIVITIES.md#where-the-west-wind-blows)                     |
+| 9:50 AM  | [Energizer 2](ACTIVITIES.md#i-love-my-neighbor-who)                        |
+| 10:00 AM | [Breakfast](ACTIVITIES.md#breakfast)                                       |
+| 10:30 AM | [Introduction talk](ACTIVITIES.md#introduction-talk)                       |
+| 10:40 AM | [Portfolio workshop](ACTIVITIES.md#portfolio)                              |
+| 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                               |
+| 9:50 AM  | [Energizer 3](ACTIVITIES.md#evolution-rock-paper-scissors)                 |
+| 2:00 PM  | [Brainstorming](ACTIVITIES.md#brainstorming)                               |
+| 2:00 PM  | [Forming groups](ACTIVITIES.md#forming-groups)                             |
+| 2:05 PM  | [Yes-and sessions](ACTIVITIES.md#yes-and-sessions)                         |
+| 2:10 PM  | [Running a brainstorm session](ACTIVITIES.md#running-a-brainstorm-session) |
+| 2:15 PM  | [After a brainstorm](ACTIVITIES.md#after-a-brainstorm)                     |
+| 2:20 PM  | [Project time](ACTIVITIES.md#freeform-projects)                            |
+|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)                   |
+|          | [Engagement](ACTIVITIES.md#engagement)                                     |
+|          | [Facilitating](ACTIVITIES.md#facilitating)                                 |
+|          | [Answering questions](ACTIVITIES.md#answering-questions)                   |
```

Comment:
Would it make sense to have a time scheduled for that? Perhaps just link to
"Project time" and in the project time guidelines have an index at the top?

Ex.

- - -

# Project Time

- [Scoping and concept](ACTIVITIES.md#scoping-and-concept)
- [Engagement](ACTIVITIES.md#engagement)
- [Facilitating](ACTIVITIES.md#facilitating)
- [Answering questions](ACTIVITIES.md#answering-questions)

On Fri, Aug 7, 2015 at 11:50 AM, Max Wofford <notifications@github.com>
wrote:

> In cohort_4/playbook/SCHEDULE.md
> <https://github.com/hackedu/hack-camp/pull/366#discussion_r36549345>:
>
> > +| 9:50 AM  | [Energizer 2](ACTIVITIES.md#i-love-my-neighbor-who)                        |
> > +| 10:00 AM | [Breakfast](ACTIVITIES.md#breakfast)                                       |
> > +| 10:30 AM | [Introduction talk](ACTIVITIES.md#introduction-talk)                       |
> > +| 10:40 AM | [Portfolio workshop](ACTIVITIES.md#portfolio)                              |
> > +| 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                               |
> > +| 9:50 AM  | [Energizer 3](ACTIVITIES.md#evolution-rock-paper-scissors)                 |
> > +| 2:00 PM  | [Brainstorming](ACTIVITIES.md#brainstorming)                               |
> > +| 2:00 PM  | [Forming groups](ACTIVITIES.md#forming-groups)                             |
> > +| 2:05 PM  | [Yes-and sessions](ACTIVITIES.md#yes-and-sessions)                         |
> > +| 2:10 PM  | [Running a brainstorm session](ACTIVITIES.md#running-a-brainstorm-session) |
> > +| 2:15 PM  | [After a brainstorm](ACTIVITIES.md#after-a-brainstorm)                     |
> > +| 2:20 PM  | [Project time](ACTIVITIES.md#freeform-projects)                            |
> > +|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)                   |
> > +|          | [Engagement](ACTIVITIES.md#engagement)                                     |
> > +|          | [Facilitating](ACTIVITIES.md#facilitating)                                 |
> > +|          | [Answering questions](ACTIVITIES.md#answering-questions)                   |
>
> How can I schedule time for Facilitating vs Answering questions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hack-camp/pull/366/files#r36549345>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us